internal/service/workers/nooneisforgotten/main.go

```diff
@@ -55,6 +63,87 @@ func updatePassportScanEvents(db *pgdb.DB) error {
 	return nil
 }
 
+func claimPassportScanEvents(cfg config.Config) error {
+	evType := cfg.EventTypes().Get(evtypes.TypePassportScan, evtypes.FilterInactive)
+	if evType == nil {
+		return nil
+	}
+
+	db := cfg.DB().Clone()
+	events, err := pg.NewEvents(db).FilterByType(evtypes.TypePassportScan).FilterByStatus(data.EventFulfilled).Select()
```

Comment:
it is now repeated so many times that it's better to move to struct, fix only if it is fast